kpm/lib/kpm/tasks.rb

```diff
@@ -432,6 +432,97 @@ def system
                              options[:killbill_web_path])
         end
 
+        method_option :export,
+                      :type    => :string,
+                      :default => nil,
+                      :desc    => 'export account for a provided id.'
+        method_option :import,
+                      :type    => :string,
+                      :default => nil,
+                      :desc    => 'import account for a previously exported file.'
+        method_option :tenant_record_id,
+                      :type    => :numeric,
+                      :default => nil,
+                      :desc    => 'replace the tenant_record_id before importing data.'
+        method_option :account_record_id,
+                      :type    => :numeric,
+                      :default => nil,
+                      :desc    => 'replace the account record_id before importing data.'
+        method_option :skip_payment_methods,
+                      :type    => :boolean,
+                      :default => false,
+                      :desc    => 'Skip or swap payment types other than __EXTERNAL_PAYMENT__.'
+        method_option :config_file,
+                      :type    => :string,
+                      :default => nil,
+                      :desc    => 'Yml that contains killbill api connection and DB connection'
+        method_option :killbill_api_credentials,
+                      :type    => :array,
+                      :default => nil,
+                      :desc    => 'Killbill api credentials <api_key> <api_secrets>'
+        method_option :killbill_credentials,
+                      :type    => :array,
+                      :default => nil,
+                      :desc    => 'Killbill credentials <user> <password>'
+        method_option :killbill_url,
+                      :type    => :string,
+                      :default => nil,
+                      :desc    => 'Killbill URL ex. http://127.0.0.1:8080'
+        method_option :database,
+                      :type    => :array,
+                      :default => nil,
+                      :desc    => 'DB connection parameters <database> <user> <password>'
+        desc 'account', 'export/import accounts'
+        def account
+          puts "Please wait processing the request!!!\n"
```

Comment:
It would be good to have some progress displayed instead, maybe each time a new table is loaded for instance?